contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -1413,6 +1413,71 @@ boolean CreateProcessW(String lpApplicationName, char[] lpCommandLine,
      */
     boolean GetVersionEx(OSVERSIONINFOEX lpVersionInfo);
 
+    /**
+     * The VerifyVersionInfoW function compares a set of operating system
+     * version requirements to the corresponding values for the currently
+     * running version of the system.This function is subject to manifest-based
+     * behavior.
+     * 
+     * @param lpVersionInformation
+     *            A pointer to an OSVERSIONINFOEX structure containing the
+     *            operating system version requirements to compare. The
+     *            dwTypeMask parameter indicates the members of this structure
+     *            that contain information to compare.
+     * 
+     *            You must set the dwOSVersionInfoSize member of this structure
```

Comment:
you need to add p-tags to get the paragraph formatted (javadoc is HTML). The newlines are removed when rendered to HTML.